packages/website/src/pages/docs/Guides/Language.mdx

```diff
@@ -50,12 +167,46 @@ d.dist`}
 f`}
 />
 
+There's no `return` statement in Squiggle; function definitions are values.
+
+If you need to define local variables in functions, you can use blocks:
+
+<SquiggleEditor
+  defaultCode={`mul6(x) = {
+  t = x * 2
+  t * 3
+}`}
+/>
+
 ### Anonymous Functions
 
-`f(x) = x*x` and `f = {|x| x*x}` are equivalent.
+`f(x, y) = x * y` and `f = {|x, y| x * y}` are equivalent.
+
+Squiggle functions are values, and you can save them to variables, pass them to other functions as arguments, return functions from functions, and so on.
 
 <SquiggleEditor defaultCode={`{|t| normal(t^2, t^1.2+.01)}`} />
 
+### Viewing functions
+
+Squiggle viewer supports two types of functions:
+
+1. `(number) => number`
+2. `(number) => distribution`
+
+<SquiggleEditor defaultCode={`numberToNumber(x) = x * x
+numberToDist(x) = normal(x + 1, 3)
+otherFunction(x, y) = x + y`} />
+
+When Squiggle displays a single parameter function, it needs to select some range of parameter values.
+
+The default range is `0...10`.
+
+The default parameter value range can be overridden in several ways:
+
+- With `Plot.numericFn` or `Plot.distFn` plots (look for `xScale` parameter)
+- Through the chart's settings in the UI (look for a gear icon next to the variable name)
+- By utilizing annotations (explained below)
```

Comment:
``With parameter annotations (explained below)``

Reply:
"Utilizing" is a weird word imo

Reply:
It was suggested by ChatGPT :) It often offers rewrites with more fancy / less popular words (usually bad for documentation, because some readers can be non-native English speakers), I usually declined those suggestions but wasn't sure in this case.

<img width="848" alt="Captura de pantalla 2023-07-26 a la(s) 13 45 01" src="https://github.com/quantified-uncertainty/squiggle/assets/89368/ce7547ee-0f21-4896-95a0-9741d03fc8bd">